Author: Gregor Riepl <onitake@gmail.com>
Description: Removes cmake scripts installation rule.
  CMAKE_INSTALL_DATADIR is a architecture- and cmake package
  dependent location. On top of that, these scripts are
  highly specific to Uranium.
  Instead, we install them to /usr/share/Uranium/cmake via
  a custom debian/.install file.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,8 +48,6 @@
 else()
     install(DIRECTORY UM DESTINATION lib/python${PYTHON_VERSION_MAJOR}/dist-packages)
 endif()
-install(FILES ${CMAKE_SOURCE_DIR}/cmake/UraniumTranslationTools.cmake
-        DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/Modules/ )
 install(DIRECTORY resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
 install(DIRECTORY plugins DESTINATION lib/uranium)
 
